content/en/docs/Writing policies/mutate.md

```diff
@@ -351,6 +351,190 @@ The anchor processing behavior for mutate conditions is as follows:
+            kinds:
+            - ConfigMap
+            names:
+            - dictionary-1
+            namespaces:
+            - staging
+...
+```
+
+### Variables Referencing Target Resources
+
+To reference data in target resources, you can define the variable `target` followed by the path to the desired attribute. For example, using `target.metadata.labels.env` references the label `env` in the target resource.
+
+This policy copies the configmaps' value `target.data.key` to their label with the key `env`.
+
+```yaml
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: sync-cms
+spec:
+  mutateExistingOnPolicyUpdate: false
+  rules:
+  - name: concat-cm
+    match:
+      any:
+      - resources:
+          kinds:
+          - ConfigMap
+          names:
+          - cmone
+          namespaces:
+          - foo
+    mutate:
+      targets:
+        - apiVersion: v1
+          kind: ConfigMap
+          name: cmtwo
+          namespace: bar
+        - apiVersion: v1
+          kind: ConfigMap
+          name: cmthree
+          namespace: bar
+      patchesJson6902: |-
+        - op: add
+          path: "/metadata/labels/env"
+          value: "{{ target.data.key }}"  
+```
+
+`@` is added to reference the in-line value of the target resource.
+
+This policy adds the value of `keyone` from the trigger configmap `foo/cmone` as the prefix to target configmaps in their data with `keynew`.
+
+```yaml
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: sync-cms
+spec:
+  mutateExistingOnPolicyUpdate: false
+  rules:
+  - name: concat-cm
+    match:
+      any:
+      - resources:
+          kinds:
+          - ConfigMap
+          names:
+          - cmone
+          namespaces:
+          - foo
+    mutate:
+      targets:
+        - apiVersion: v1
+          kind: ConfigMap
+          name: cmtwo
+          namespace: bar
+        - apiVersion: v1
+          kind: ConfigMap
+          name: cmthree
+          namespace: bar
+      patchStrategicMerge:
+        data:
+          keynew: "{{request.object.data.keyone}}-{{@}}"
+```
+
+Once a mutate existing policy is applied successfully, there will be an event and an annotation added to the target resource:
+
+```
+✗ kubectl describe deploy foobar
+...
+Events:
+  Type     Reason             Age                From                   Message
+  ----     ------             ----               ----                   -------
+  Normal   PolicyApplied      29s (x2 over 31s)  kyverno-mutate         policy add-sec/add-sec-rule applied
+
+✗ kubectl get deploy foobar -o yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    ...
+    policies.kyverno.io/last-applied-patches: |
+      add-sec-rule.add-sec.kyverno.io: added /spec/template/spec/containers/0/securityContext
+```
+
+To troubleshoot the policy application failure, you can inspect `UpdateRequest` custom resource to get details. Succeeded updaterequests are automatically cleaned up by Kyverno.
```

Comment:
"...you can inspect the `UpdateRequest` Custom Resource to get details. Successful UpdateRequests are automatically cleaned up by Kyverno."